hack/validation/v1beta1/kubelet.sh

```diff
@@ -0,0 +1,22 @@
+# Kubelet Validation 
```

Comment:
Do we really need to duplicate these changes or can we find a way to parameterize and reduce the duplication?